pom.xml

```diff
@@ -292,7 +292,6 @@
                 <configuration>
                     <compilerArgs>
                         <arg>-Xlint:all,-try</arg>
-                        <arg>-Werror</arg>
```

Comment:
This flag is causing the build to fail due to a deprecation warning. Not sure if there's a more elegant way to avoid the warning other than switching to the new builder (which looks complicated)